extensions/azurecore/src/azureResource/commands.ts

```diff
@@ -73,7 +75,7 @@ export function registerAzureResourceCommands(appContext: AppContext, tree: Azur
 	// Resource Tree commands
 
 	vscode.commands.registerCommand('azure.resource.selectsubscriptions', async (node?: TreeNode) => {
-		if (!(node instanceof AzureResourceAccountTreeNode)) {
+		if (!(node instanceof AzureResourceAccountTreeNode) && !(node instanceof FlatAccountTreeNode)) {
```

Comment:
Would it make sense to add an array of node types that need to be excluded here? The list will only seem to increase in size

Reply:
the old view will be removed once this work is all done, so actually there will be one type here.